convert/src/label_studio_to_badgerdoc/models/label_studio_models.py

```diff
@@ -20,7 +20,7 @@ class LabelStudioRequest(BaseModel):
     input_annotation: S3Path
     output_bucket: str
     validation_type: ValidationType
-    deadline: Optional[datetime]
+    deadline: Optional[datetime] = None
```

Comment:
There's no point of adding "= None" as soon as "Optional[datetime] does the same.
It also applies to all other chenges in this commit

Reply:
pyright shows warning without " = None"